services/src/main/java/org/keycloak/services/resources/admin/RealmLocalizationResource.java

```diff
@@ -159,10 +159,6 @@ public Stream<String> getRealmLocalizationLocales() {
     @Operation()
     public Map<String, String> getRealmLocalizationTexts(@PathParam("locale") String locale,
             @Deprecated @QueryParam("useRealmDefaultLocaleFallback") Boolean useFallback) {
-        if (!AdminPermissions.realms(session, auth.adminAuth()).isAdmin()) {
-            throw new ForbiddenException();
-        }
-
```

Comment:
After this change, the test in `PermissionsTest#localizations()` fails. 

I see that the other methods have `auth.requireAnyAdminRole();` instead - would this work here as well instead of removing the check altogether?

Unfortunately the PR doesn't link an issue, and the PR doesn't have a good reproducer I can use to test this interactively.